conductr_cli/bndl_main.py

```diff
@@ -75,6 +75,11 @@ def build_parser():
                         help='Description to use for the generated ConductR component',
                         default='')
 
+    parser.add_argument('--annotation',
+                        nargs=2,
+                        action='append',
+                        dest='annotations')
+
```

Comment:
I think this should be consistent with how we assign env vars for the sandbox ie one arg and not two.